input/papers/2016/2016arXiv160908671H/tev-000095.yaml

```diff
@@ -0,0 +1,26 @@
+paper_id: 2016arXiv160908671H
+source_id: 95
+
+data:
+  livetime: 13.4 # year 2012
```

Comment:
The total livetime is 164 hours, mentioned in the Section 2 text. Please put that number here, and let's not try to put the livetimes for the subsets of data.